src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -1630,7 +1633,17 @@ static void UCS_F_ALWAYS_INLINE uct_rc_mlx5_iface_copy_iov_to_dm(
         log_sge->num_sge = 0;
     }
 
-    while (iov_iter.iov_index != iovcnt) {
+    while (iov_iter.iov_index < iovcnt) {
+        to_copy = ucs_align_down(
+                iov[iov_iter.iov_index].length - iov_iter.buffer_offset,
+                sizeof(word));
+        UCS_WORD_COPY(volatile uint64_t, dm, misaligned_t,
+                      UCS_PTR_BYTE_OFFSET(iov[iov_iter.iov_index].buffer,
+                                          iov_iter.buffer_offset),
```

Comment:
maybe use helper var (void *ptr or void *iov_ptr) ?
to save UCS_PTR_BYTE_OFFSET(...) value